api/models.py

```diff
@@ -11,6 +11,7 @@ def __str__(self):
 class Entity(BaseClass):
     location = models.CharField(max_length=120)
     user = models.ForeignKey('auth.User')
```

Comment:
You should keep in mind that we are using `auth.User` from Django as a ForeignKey in Entity.
In any case the email will be from a user, an entity can have many users and so. It is not a bad idea to consider a contact email or something from similar nature, but since we don't know if every Entity will have this field we chose not to put it.

Reply:
Well I wasn't in that conversation but I don't think that's a good workaround. In my opinion there should be an email field both on users and entities.

The reason is that users should have an email to validate register email, and the entity itself should always have an email so that they can be mailed by clients to request information about adoptions, doubts...
No¿?

Reply:
It is not a workaround, it is precisely for validation.
The entity email case you mention is a good case and can be considered, but keep in mind it can happen that an Entity doesn't have any email contact info. just a phone. 
Furthermore, if an entity already has an email for its animal shelter, put it as login email seems logical to me.
The only case I can think where it is truly needed a separated user email and entity email is when there is an entity with an email and several users that belong to a same entity, in that case I think it can be useful, but let's keep in mind that we are making models for the general, no the possibles. 
If we need to put more fields in our models we will always have time to add, but to remove can be complex in the future.